WordPress/src/main/java/org/wordpress/android/ui/reader/ReaderPostListFragment.java

```diff
@@ -1315,6 +1325,11 @@ private void submitSearchQuery(@NonNull String query) {
         hideSearchMessage();
         hideSearchSuggestions();
 
+        if (!NetworkUtils.isNetworkAvailable(getContext())) {
+            resetPostAdapter(mPostListType);
```

Comment:
Why did you add `resetPostAdapter(mPostListType)` here? I removed that line and couldn't find any difference on testing.